compiler/optimizer/VPConstraint.hpp

```diff
@@ -934,7 +934,7 @@ class VPKnownObject : public TR::VPFixedClass
 
    protected:
    VPKnownObject(TR_OpaqueClassBlock *klass, TR::Compilation * comp, TR::KnownObjectTable::Index index, bool isJavaLangClass)
-      : TR::VPFixedClass(klass, comp, KnownObjectPriority), _index(index), _isJavaLangClass(isJavaLangClass) {}
+      : TR::VPFixedClass(klass, comp, KnownObjectPriority), _index(index), _isJavaLangClass(isJavaLangClass), _isArrayWithConstantElements(false) {}
```

Comment:
I am not sure I'm a fan of hard coding this to false - I could see a default parameter to say false unless you say otherwise? @jdmpapin thoughts since you are familiar with this constraint.

Reply:
Since constraints are supposed to be immutable (like you mentioned elsewhere), it will need to be provided when the constraint is constructed